annotations.md

```diff
@@ -40,6 +40,13 @@ This specification defines the following annotation keys, intended for but not l
     ```
 * **org.opencontainers.image.title** Human-readable title of the image (string)
 * **org.opencontainers.image.description** Human-readable description of the software packaged in the image (string)
+* **org.opencontainers.image.base.digest** [Digest](descriptor.md#digests) of the image this image is based on (string)
```

Comment:
👍 for the single reference. I'm not fully sure what descriptor means here, and how that is a string. I like the descriptor reference, just not sure what the value looks like. Is it a json string, in the form of a descriptor? Perhaps we add some examples where needed.

Reply:
It's not a descriptor, it's a _digest_, the definition for which is part of descriptor.md, here: https://github.com/opencontainers/image-spec/blob/master/descriptor.md#digests

I think that's just raw markdown making this sort of hard to read.